tracer/test/Datadog.Trace.Tests/Telemetry/config_norm_rules.json

```diff
@@ -398,6 +398,8 @@
   "DD_CIVISIBILITY_FLAKY_RETRY_COUNT": "ci_visibility_flaky_retry_count",
   "DD_CIVISIBILITY_TOTAL_FLAKY_RETRY_COUNT": "ci_visibility_total_flaky_retry_count",
   "DD_CIVISIBILITY_IMPACTED_TESTS_DETECTION_ENABLED": "dd_civisibility_impacted_tests_detection_enabled",
+  "DD_TEST_MANAGEMENT_ENABLED": "test_management_enabled",
+  "DD_TEST_MANAGEMENT_ATTEMPT_TO_FIX_RETRIES": "test_management_attempt_to_fix_retries",
```

Comment:
Don't forget the dd-go PR 🙂

Reply:
Will do that PR.

Reply:
https://github.com/DataDog/dd-go/pull/175123